simcon_project/templates/researcher_view.html

```diff
@@ -1,35 +1,33 @@
+{% extends "researcher_base.html" %}
+{% load render_table from django_tables2 %}
+{% load static %}
+{% block nav-researcherView %}{% endblock %} 
+{% block content %}
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta name="description" content="Created for Simulated Conversations Senior Capstone Project">
-
-    <!-- prevent line breaks when using form fields -->
-    <style type="text/css">
-    form {display:inline; margin:0px; padding:0px;}
-    </style>
-    
-
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"> </script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+    <link rel="stylesheet" src="{% static 'users/researcherTable.css' %}">
     <title>
-    {% block title %}Simulated Conversations | Researcher{% endblock %}
+    SimCon | Researcher View
     </title>
-    
-    {% block extrahead %} {% endblock %}
   </head>
-
   <body>
-    <div class="container">
-      <h1> THIS IS THE Researcher PAGE</h1>
-        {% block content %}
-        {% endblock %}
-    </div> <!-- /container -->
-
-    <!-- Bootstrap core JavaScript
-    ================================================== -->
-
-    <!-- Placed at the end of the document so the pages load faster -->
+<div class="table-container"> 
+  {% render_table responseTable %}
+  <footer>
+  <p>Copyright 2020 Portland State University</p>
```

Comment:
This is interesting... I like it. Also, a bunch of bootstrap stuff is meant to be included at the end of the file. Should we have a footer html that will include this thing plus possibly the bootstrap stuff.... like a common footer that ppl can include?